metadata/index.html

```diff
@@ -737,7 +735,7 @@
             <dt id="table-notes"><code>notes</code></dt>
             <dd>
               <p>
-                An <a>array property</a> that provides an array of objects representing arbitrary annotations on the annotated tabular data model. This specification does not place any constraints on the structure of these objects.
+                A <a>common property</a> that provides one or more objects representing arbitrary annotations on the annotated tabular data model. This specification does not place any constraints on the structure of these objects.
```

Comment:
Good point, I'll revert this. Probably just do supimple append without comparing for duplicates, which would be "garbage in" anyway"